Crashing when trying to emit events

Thanks for making this library! I'm working on integrating this with my Go music player [Supersonic](https://github.com/dweymouth/supersonic) and while the media key commands are working, emitting events (eg `eventHandler.OnSeek`) is crashing. Any idea what might be wrong? Here's a stack trace:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x90 pc=0x5e390d]

goroutine 9 [running]:
github.com/godbus/dbus/v5.(*Conn).getSerial(...)
	/home/drew/go/pkg/mod/github.com/godbus/dbus/v5@v5.1.0/conn.go:369
github.com/godbus/dbus/v5.(*Conn).sendMessageAndIfClosed(0x0, 0xc002e257d0, 0xc002e25830?)
	/home/drew/go/pkg/mod/github.com/godbus/dbus/v5@v5.1.0/conn.go:490 +0x2d
github.com/godbus/dbus/v5.(*Conn).Emit(0x21e5c28?, {0xce1412, 0x17}, {0xc003dd3320, 0x24}, {0xc0045f6260?, 0x1, 0x1})
	/home/drew/go/pkg/mod/github.com/godbus/dbus/v5@v5.1.0/export.go:245 +0x4e5
github.com/quarckster/go-mpris-server/pkg/events.(*orgMprisMediaPlayer2PlayerEventHandler).OnSeek(0xc00036f950, 0x7ba1dd?)
	/home/drew/go/pkg/mod/github.com/quarckster/go-mpris-server@v1.0.0/pkg/events/orgMprisMediaPlayer2PlayerEventHandler.go:126 +0xd2
github.com/dweymouth/supersonic/backend.NewMPRISHandler.func1()
	/home/drew/supersonic/backend/mpris.go:49 +0x4e
github.com/dweymouth/supersonic/player.(*Player).eventHandler(0xc0000d1440, {0xe0ed10, 0xc0000a4870})
	/home/drew/supersonic/player/player.go:648 +0x134
created by github.com/dweymouth/supersonic/player.(*Player).Init
	/home/drew/supersonic/player/player.go:181 +0x36e
```

--- a/pkg/events/orgMprisMediaPlayer2EventHandler.go
+++ b/pkg/events/orgMprisMediaPlayer2EventHandler.go
@@ -34,7 +34,7 @@ func allRootProps(adapter types.OrgMprisMediaPlayer2Adapter) []string {
 
 func newOrgMprisMediaPlayer2EventHandler(mpris *server.Server) *orgMprisMediaPlayer2EventHandler {
 	eventHandler := orgMprisMediaPlayer2EventHandler{
-		conn:     mpris.Conn,
+		mpris:    mpris,
 		iface:    "org.mpris.MediaPlayer2",
 		adapter:  mpris.RootAdapter,
 		allProps: allRootProps(mpris.RootAdapter),
@@ -43,7 +43,7 @@ func newOrgMprisMediaPlayer2EventHandler(mpris *server.Server) *orgMprisMediaPla
 }
 
 type orgMprisMediaPlayer2EventHandler struct {
-	conn     *dbus.Conn
+	mpris    *server.Server
 	iface    string
 	adapter  types.OrgMprisMediaPlayer2Adapter
 	allProps []string
@@ -54,7 +54,7 @@ func (o *orgMprisMediaPlayer2EventHandler) EmitChanges(props []string) error {
 	if err != nil {
 		return err
 	}
-	return internal.EmitPropertiesChanged(o.conn, o.iface, changes)
+	return internal.EmitPropertiesChanged(o.mpris.Conn, o.iface, changes)
 }
 
 func (o *orgMprisMediaPlayer2EventHandler) OnAll() error {

--- a/pkg/events/orgMprisMediaPlayer2PlayerEventHandler.go
+++ b/pkg/events/orgMprisMediaPlayer2PlayerEventHandler.go
@@ -58,7 +58,7 @@ func newOrgMprisMediaPlayer2PlayerEventHandler(
 	mpris *server.Server,
 ) *orgMprisMediaPlayer2PlayerEventHandler {
 	eventHandler := orgMprisMediaPlayer2PlayerEventHandler{
-		conn:          mpris.Conn,
+		mpris:         mpris,
 		iface:         "org.mpris.MediaPlayer2.Player",
 		adapter:       mpris.PlayerAdapter,
 		allProps:      allPlayerProps(mpris.PlayerAdapter),
@@ -81,7 +81,7 @@ func newOrgMprisMediaPlayer2PlayerEventHandler(
 }
 
 type orgMprisMediaPlayer2PlayerEventHandler struct {
-	conn             *dbus.Conn
+	mpris            *server.Server
 	iface            string
 	adapter          types.OrgMprisMediaPlayer2PlayerAdapter
 	allProps         []string
@@ -99,7 +99,7 @@ func (o *orgMprisMediaPlayer2PlayerEventHandler) EmitChanges(props []string) err
 	if err != nil {
 		return err
 	}
-	return internal.EmitPropertiesChanged(o.conn, o.iface, changes)
+	return internal.EmitPropertiesChanged(o.mpris.Conn, o.iface, changes)
 }
 
 func (o *orgMprisMediaPlayer2PlayerEventHandler) OnEnded() error {
@@ -123,7 +123,7 @@ func (o *orgMprisMediaPlayer2PlayerEventHandler) OnTitle() error {
 }
 
 func (o *orgMprisMediaPlayer2PlayerEventHandler) OnSeek(position types.Microseconds) error {
-	o.conn.Emit("/org/mpris/MediaPlayer2", o.iface+".Seeked", int64(position))
+	o.mpris.Conn.Emit("/org/mpris/MediaPlayer2", o.iface+".Seeked", int64(position))
 	return o.EmitChanges(o.onSeekProps)
 }
 

